frontend/src/views/ClientListView.vue

```diff
@@ -4,15 +4,21 @@ import Card from "primevue/card";
 import InputText from "primevue/inputtext";
 import { onMounted } from "vue";
 import { ref } from "vue";
+import { useRouter } from "vue-router";
 
-import { getClients } from "@/backendClient";
+import { deleteClient, getClients } from "@/backendClient";
 import type { Client } from "@/global/types/entities";
 import { ROUTES } from "@/router";
 
+const router = useRouter();
 const clientsList = ref<Client[]>([]);
-
 const value = ref(null);
 
+// we should provide an argument in the place where we call the function
+async function removeClient(client: Client) {
+  await deleteClient(client.id);
```

Comment:
Looks like the deletion is failing because of a foreign-key constraint (i.e. cannot delete a client with open orders).
Should we add error handling?